### PR TITLE
added ch.are.windenergie-bundesinteressen

### PR DESCRIPTION
**Pull request by rebert**
[try](https://mf-chsdi3.int.bgdi.ch/ltrea_bund_wind/shorten/73ae6f6ea3)

### CODE REVIEW

**Comment by rebert** [comment]
Add colors of Legend when medium is print

**Comment by AFoletti** [comment]
It looks good to me! Let's wait for Datenherr feedback

File: chsdi/templates/htmlpopup/windenergie_bundesinteressen.mako [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+       <tr class="wind_legend">
+           <td><div class="wind_kbik_main wind_kbik_4"></div><p>${_('ch.are.windenergie-bundesinteressen.bik4')}</p></td>
+           <td><p>${desc_bik4}</p></td>
+       </tr>
+       <tr class="wind_legend">
+           <td><div class="wind_kbik_main wind_kbik_5"></div><div class="wind_kbik_main wind_kbik_1"></div><p>${_('ch.are.windenergie-bundesinteressen.bik5')}, ${_('ch.are.windenergie-bundesinteressen.bik1')}</p></td>
+           <td><p>${desc_bik1}</p></td>
+       </tr>
+       <tr class="wind_info">
+           <td colspan="2"><p>${_('ch.are.windenergie-bundesinteressen.tt_info_footer')} <a href="http://www.are.admin.ch/windenergie" target="_blank">www.are.admin.ch/windenergie</a></p></td>
```
Comment (loicgasser) [inline chsdi/templates/htmlpopup/windenergie_bundesinteressen.mako]: they support https and even make a redirect, so you can change it I think.

**Comment by AFoletti** [comment]
I just found a bug:
if you click on a jucnction between multiple colors (in order to get 4 tooltips) you correctly get 4 tootlips, but the displayed info is always the same (only 1 color, same descrption)
Sorry for the lame explanation. I'll show you what I mean if need be

**Comment by rebert** [comment]
@loicgasser please review and merge

**Pull request merged**